test_conformance/geometrics/test_geometrics_base.h

```diff
@@ -0,0 +1,481 @@
+//
+// Copyright (c) 2022 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifndef _TEST_GEOMETRICS_BASE_H
+#define _TEST_GEOMETRICS_BASE_H
+
+#include <vector>
+#include <map>
+#include <memory>
+#include <CL/cl_half.h>
+
+#include "testBase.h"
+#include <limits>
+#include <cmath>
+
+#define HALF_P_NAN 0x7e00
+#define HALF_N_NAN 0xfe00
+#define HALF_P_INF 0x7c00
```

Comment:
These INF values are only used in 1 pace, so might be easier to just inline them

Reply:
Done